pkg/fixtures/variadic.go

```diff
@@ -0,0 +1,7 @@
+package test
+
+type VariadicFunction = func(args1 string, args2 ...interface{}) interface{}
+
+type Variadic interface {
+	VariadicFunction(str string, vFunc VariadicFunction) error
```

Comment:
I don't think this is testing the original issue. 

https://github.com/vektra/mockery/issues/431

The issue was with methods that return a function in which one of the arguments is variadic:

```
LogMethodf(level Level) func(message string, a ...interface{})
```

Reply:
I just tested It and It looks like it works, I will push a change to show It

Reply:
Oh sorry I misread this. Yes this is good 👍🏻